types/interfaces/AssetError.ts

```diff
@@ -0,0 +1,4 @@
+export declare interface AssetError {
+  type: 'invalid_input' | string;
+  messages: string[];
```

Comment:
I don't have a strong opinion either way on this, but should we be consistent with our use of `Array<string>` and `string[]`?